pkgs/tools/misc/rmate-sh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rmate";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "aurora";
+    repo = "rmate";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-fmK6h9bqZ0zO3HWfZvPdYuZ6i/0HZ1CA3FUnkS+E9ns=";
+  };
+
+  buildPhase = ''
+    true
+  '';
```

Comment:
```suggestion
  dontBuild = true;
```

Reply:
Thanks! I tried `doBuild = false;`, as with `doCheck`, but it didn't work.